src/utils/tranche.utils.ts

```diff
@@ -1,8 +1,10 @@
-import { DAYS_IN_YEAR } from "@consts";
+import { DAYS_IN_YEAR, DefaultTranche, TrancheDataElement } from "@consts";
 import { Fraction, RepaymentSchedule, Tranche } from "@credix/credix-client";
+import { TrancheFormValue, TrancheName, TrancheTitle } from "@credix_types/tranche.types";
 import { TokenAmount } from "@solana/web3.js";
 import Big from "big.js";
-import { round } from "./format.utils";
+import { capitalizeFirstLetter, round } from "./format.utils";
+import { parse } from "csv-parse/sync";
```

Comment:
I believe this is the Node version and we will be running in the browser.
https://csv.js.org/parse/distributions/nodejs_esm/
vs
https://csv.js.org/parse/distributions/browser_esm/